template/android/app/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
+
+# Define the library name here.
+project(rndiffapp_appmodules)
```

Comment:
We probably want something else in here, instead of `rndiffapp`.

Reply:
Thanks! Nice find 👍